wtransport/Cargo.toml

```diff
@@ -32,6 +32,7 @@ required-features = ["self-signed"]
 bytes = "1.4.0"
 pem = "3.0.4"
 quinn = { version = "0.11.3", default-features = false, features = ["runtime-tokio", "rustls"] }
+quinn-proto = { version = "0.11.6", default-features = false, features = ["rustls"] }
```

Comment:
I was initially confused about why this would be a new import statement; then I realized `quinn` re-exports [everything I need from `quinn-proto`](https://github.com/quinn-rs/quinn/blob/bb02a12a8435a7732a1d762783eeacbb7e50418e/quinn/src/lib.rs#L64-L68) already. I refactored my app and no longer need quinn-proto (or quinn) dependencies.

Reply:
Ok (to avoid useless code, I'd avoid re-exporting `quinn-proto`)